crates/api/src/arc.rs

```diff
@@ -0,0 +1,305 @@
+//! Kitsune2 arcs represent a range of hash locations on the DHT.
+//!
+//! An arc can exist in its own right and refer to a range of locations on the DHT.
+//! It can also be used in context, such as an agent. Where it represents the range of locations
+//! that agent is responsible for.
+
+use serde::{Deserialize, Serialize};
+
+/// The definition of a storage arc compatible with the concept of
+/// storage and querying of items in a store that fall within that arc.
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Default)]
+#[serde(untagged)]
+pub enum StorageArc {
```

Comment:
We're talking about locations on the DHT, so how about
```suggestion
pub enum DhtArc {
```

Reply:
I see this has already been discussed.

Reply:
I'd be happy with this name too, any other votes either way?

Reply:
If not, I'll just do the rename :)